test/tests/problems/eigen_problem/tests

```diff
@@ -49,6 +49,8 @@
     cli_args = 'Executioner/eigen_solve_type=NONLINEAR_POWER Outputs/file_base=nonlinear_power'
     csvdiff = 'nonlinear_power_eigenvalues_0001.csv'
     slepc_version = '>=3.7.4'
+    # we want to test against PETSc master
```

Comment:
cpt. Obvious, remove this comment - you are not using this below either. The meaning is documented in test harness.